docs_user/modules/proc_preparing-block-storage-service-by-customizing-configuration.adoc

```diff
@@ -13,16 +13,16 @@ The high level explanation of the tailor-made approach is:
 endif::[]
```

Comment:
1. Is the purpose of this procedure to “customize” the configuration? 
2. If yes, are users “customizing” the configuration so that they only migrate certain aspects of the configuration to RHOSO?

Reply:
In the older OpenStack we would have the same `cinder.conf` file for ALL the services, whereas in the new OpenStack we lean towards individual service configuration.

So basically this "customization" is just a slice & dice process to convert the single file old configuration and split it into the configuration of the different Cinder services.  It also covers a couple of things that are done differently in the new OpenStack (like the vendor images and adding files).